addons/project/static/src/components/project_task_state_selection/project_task_state_selection.scss

```diff
@@ -1,4 +1,7 @@
 .o_field_project_task_state_selection{
+    // FIXME: ugly override to align icons in list view.
+    margin-top: var(--fieldTaskStateSelection-marginn);
```

Comment:
Shouldn't work as... there is a typo (double "n" at "margin"). :upside_down_face:

Reply:
Uh oh sorry for this 😅